.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - "8"
```

Comment:
Node 8 was EOL on December 31, 2019, can we remove it?

Reply:
@dlongley @gannan08  @davidlehn  can we get rather quick conesnsus on this: should travis files be upgraded to test:

```
node_js:
  - "10"
  - "12"
  - "13"
  - "node"
```

so to ensure we are testing only in currently supported node versions?

I think @mattcollier has a good point here.

Reply:
+1 agree

Reply:
Addressed here: https://github.com/digitalbazaar/http-signature-zcap-verify/pull/9/commits/a0480496eb65006c6f7bda967660ee32cf540272

I also did a quick test using nvm and there are no breaking changes from 10-13 so we are good to go.